apps/api/src/app/logo/logo.service.ts

```diff
@@ -50,13 +49,12 @@ export class LogoService {
       abortController.abort();
     }, this.configurationService.get('REQUEST_TIMEOUT'));
 
-    return got(
+    return fetch(
       `https://t0.gstatic.com/faviconV2?client=SOCIAL&type=FAVICON&fallback_opts=TYPE,SIZE,URL&url=${aUrl}&size=64`,
       {
         headers: { 'User-Agent': 'request' },
-        // @ts-ignore
         signal: abortController.signal
       }
-    ).buffer();
+    ).then((res) => res.arrayBuffer());
```

Comment:
Seems like it breaks the platform logos for accounts. I'm trying to figure out how to replace it properly.

Reply:
OK, fixed. 

BTW. If `response.blob()` would be used, we could use both `blob.arrayBuffer()` as well as `blob.type`. `type` contains the original media type. 
The issue is that `image/png` is hardcoded: https://github.com/ghostfolio/ghostfolio/blob/46cbbd3bc4af9ed395ef81476190eb37640a83de/apps/api/src/app/logo/logo.controller.ts#L49

I found a logo that had `image/jpeg` originally. And even though it was sent with `image/png` instead, it still shows itself OK.